doc/nrf/libraries/modem/nrf_modem_lib/nrf_modem_lib_lte_net_if.rst

```diff
@@ -0,0 +1,13 @@
+.. _nrf_modem_lib_lte_net_if:
+
+Network interface driver
+########################
+
+The LTE network interface in the  :ref:`nrfxlib:nrf_modem` is a Zephyr network interface driver for the nRF91 Series cellular firmware.
+
+By enabling the LTE network interface in the Modem library, you can control the nRF91 series modem as a Zephyr network interface, for example, :c:func:`net_if_up`, :c:func:`conn_mgr_if_connect`, and :c:func:`net_if_down`.
```

Comment:
I suppose it should be like this here too. Also added a missing verb (adjust to your taste).
```suggestion
By enabling the LTE network interface in the :ref:`nrfxlib:nrf_modem`, you can control the nRF91 series modem as a Zephyr network interface, for example by calling :c:func:`net_if_up`, :c:func:`conn_mgr_if_connect`, and :c:func:`net_if_down`.
```